doc/source/releases.md

```diff
@@ -1,6 +1,58 @@
 # MDAnalysis Release Notes
 
 
+## Release 2.7.0 of MDAnalysis
+
+This a minor release of MDAnalysis.
+
+This release of MDAnalysis is packaged under a [GPLv3+ license](https://www.gnu.org/licenses/gpl-3.0.en.html), additionally all contributions made from commit 44733fc214dcfdcc2b7cb3e3705258781bb491bd onwards are made under the [LGPLv2.1+ license](https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html).

```

Comment:
```suggestion
This release of MDAnalysis is packaged under a [GPLv3+ license](https://www.gnu.org/licenses/gpl-3.0.en.html), additionally all contributions made from commit `44733fc214dcfdcc2b7cb3e3705258781bb491bd` onwards are made under the [LGPLv2.1+ license](https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html).

```

Reply:
Because these notes are auto generated, I would prefer not fixing test here. If this is something that definitely should be fixed I'll go back and fix the release notes on the MDA repo.

Reply:
This was just a nitpick, hence the approval. It can be safely ignored, although it would give a slightly better rendering IMO.

Reply:
Ok, I'll fix it upstream, it shouldn't really break anything. The aim of this is for the changelog to be more readable anyways.

Reply:
Fixed upstream, just need to re-generate